### PR TITLE
First draft of shadow piercing update

### PR DESCRIPTION
**Pull request by robdodson**
This is ready for review. Note there's a TBD link to as an of yet unpublished Polymer blog post that should be going out shortly. That blog post will need to launch before this PR can be merged.

@arthurevans FYI

### CODE REVIEW

**Comment by robdodson** [comment]
oh oops! just realized i need to include an image :P

**Comment by robdodson** [comment]
@ebidel changed component to Blink>DOM after the bot complained, but crstatus reports Blink>DOM>ShadowDOM as the component. Should I stick with Blink>DOM or is there some way to teach the bot about Blink>DOM>ShadowDOM?

**Comment by robdodson** [comment]
@petele this is ready PTAL

**Comment by petele** [comment]
@Meggin - can you take a swing through this please?

File: src/content/en/updates/2017/10/remove-shadow-piercing.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+{% include "web/_shared/contributors/robdodson.html" %}
+
+Starting in Chrome 63, the shadow-piercing selectors `::shadow` and `/deep/`
+will no longer be able to style content inside of a shadow root.
+
+- The `/deep/` combinator will act as a descendant selector. `x-foo /deep/ div`
+will work like `x-foo div`.
+- The `::shadow` pseudo-element will not match any elements.
+
+Note: If your site uses Polymer, the team has put together [a thorough guide](https://www.polymer-project.org/blog/2017-10-18-upcoming-changes.html)
```
Comment (Meggin) [inline src/content/en/updates/2017/10/remove-shadow-piercing.md]: This link isn't working for me in the staging server. Might be just the nature of the staging server, but I'm also wondering if maybe notes have specific linking requirements?

File: src/content/en/updates/2017/10/remove-shadow-piercing.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+walking through steps to migrate off of `::shadow` and `/deep/`.
+
+## The decision to remove
+
+The `::shadow` and `/deep/` selectors have been deprecated in Chrome since
```
Comment (Meggin) [inline src/content/en/updates/2017/10/remove-shadow-piercing.md]: 'were deprecated in Chrome 45', is a bit leaner than 'have been deprecated in Chrome since 45'.

File: src/content/en/updates/2017/10/remove-shadow-piercing.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+The [CSS Shadow Parts](https://tabatkins.github.io/specs/css-shadow-parts/) spec
+is being advanced as an alternative to shadow piercing selectors. Shadow Parts
+will allow a component author to expose named elements in a way that preserves
+encapsulation and still allows page authors the ability to style multiple
+properties at once.
+
+## What should I do if my site uses ::shadow and /deep/?
+
+The `::shadow` and `/deep/` selectors only affect legacy Shadow DOM v0
+components. If you're using Shadow DOM v1 you should not need to change anything
```
Comment (Meggin) [inline src/content/en/updates/2017/10/remove-shadow-piercing.md]: small nit-- comma after v1.

File: src/content/en/updates/2017/10/remove-shadow-piercing.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+on your site.
+
+You can use [Chrome Canary](https://www.google.com/chrome/browser/canary.html)
+to verify your site does not break with these new changes. If you do notice
```
Comment (Meggin) [inline src/content/en/updates/2017/10/remove-shadow-piercing.md]: small nit-- remove ''do" (so just 'If you notice issues').

File: src/content/en/updates/2017/10/remove-shadow-piercing.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+issues, see if you can remove any usage of `::shadow` and `/deep/`. If not
```
Comment (Meggin) [inline src/content/en/updates/2017/10/remove-shadow-piercing.md]: small nit -- instead of 'see if you can remove any usage of' to 'try and remove any usage of'.

File: src/content/en/updates/2017/10/remove-shadow-piercing.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+relying on these selectors is too onerous, you may also consider switching from
```
Comment (Meggin) [inline src/content/en/updates/2017/10/remove-shadow-piercing.md]: Strange sentence 'if not relying on these selectors is too onerous...'

How about 'If it's too difficult to remove usage of these selectors, consider switching from native...'

File: src/content/en/updates/2017/10/remove-shadow-piercing.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+{# wf_tags: webcomponents,shadowdom,style,css,deprecations,removals #}
+{# wf_featured_image: /web/updates/images/generic/styles.png #}
+{# wf_featured_snippet: Say goodbye to shadow-piercing CSS selectors. #}
+{# wf_blink_components: Blink>DOM #}
+
+# Removing ::shadow and /deep/ in Chrome 63 {: .page-title }
+
+{% include "web/_shared/contributors/robdodson.html" %}
+
+Starting in Chrome 63, the shadow-piercing selectors `::shadow` and `/deep/`
```
Comment (Meggin) [inline src/content/en/updates/2017/10/remove-shadow-piercing.md]: Maybe reword a bit... 'Starting in Chrome 63, you cannot use the shadow-piercing selectors .... to style content inside of a shadow root'.
